src/localStorage.js

```diff
@@ -1,4 +1,18 @@
-const list = JSON.parse(localStorage.getItem('list')) || [];
+// const list = JSON.parse(localStorage.getItem('list')) || [];
```

Comment:
[Optional][Highly Recommended]

- I would recommend removing the commented code. As this is a small change, I am approving this request but please do this change before merging the pull request.